api/views/session_views.py

```diff
@@ -337,3 +337,19 @@ def post(self, request, recording_id):
             return Response({"error": "Recording not found."}, status=status.HTTP_404_NOT_FOUND)
         except Exception as e:
             return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    @extend_schema(
+        operation_id="get_lecture_summary",
+        summary="Get all lecture summaries from a recording id",
+        description="Get a summary for a specific recording using the recording ID.",
+        request=LectureSummarySerializer,
+        responses={
+            200: OpenApiTypes.OBJECT,  # Successful retrieval of data.
```

Comment:
There should be a serializer here that describes a structure that matches this:

![image](https://github.com/user-attachments/assets/2d889a10-1c49-483e-9ba8-aff7b0f00357)